src/OrbitClient.php

```diff
@@ -144,6 +150,11 @@ public function getOptions()
         return $this->options;
     }
 
+    public function setFlushCacheItems(bool $flushCacheItems): void
```

Comment:
This is a method in `CacheInterface`. Does that mean that this class should now implement `CacheInterface`?